docker/test/base/setup_export_logs.sh

```diff
@@ -126,11 +126,14 @@ function setup_logs_replication
     # It's doesn't make sense to try creating tables if SYNC fails
     echo "SYSTEM SYNC DATABASE REPLICA default" | clickhouse-client "${CONNECTION_ARGS[@]}" || return 0
 
+    debug_or_sanitizer_build=$($CLICKHOUSE_CLIENT -q "WITH ((SELECT value FROM system.build_options WHERE name='BUILD_TYPE') AS build, (SELECT value FROM system.build_options WHERE name='CXX_FLAGS') as flags) SELECT build='Debug' OR flags LIKE '%fsanitize%' OR hasThreadFuzzer()")
+    echo "Build is debug or sanitizer: $debug_or_sanitizer_build"
+
     # For each system log table:
     echo 'Create %_log tables'
     clickhouse-client --query "SHOW TABLES FROM system LIKE '%\\_log'" | while read -r table
     do
-        if [[ "$table" = "trace_log" ]]
+        if [[ "$table" = "trace_log" ]] && [[ "$debug_or_sanitizer_build" = 0 ]]
```

Comment:
But a few lines below it will attempt to create a table with identical names in both cases.

Reply:
Don't see where, this condition just adjusts list of columns, and the table is created only once.

Reply:
We are creating tables with different structures depending on this condition.
But in both cases, the name of the table will be the same.
It means - which table will be created depends on which CI task will run first.

Reply:
Ok, now I see, sigh. CREATE TABLE in the remote clickhouse cluster should not be modified, only MV on the local clickhouse from the test should. Fixed.